webrtc-stats.html

```diff
@@ -118,51 +118,48 @@
-            time for when these values are current. Most stats objects are related one-on-one to an
-            object defined in the WebRTC API; they may be thought of as being internal properties
-            of those objects.
+            An internal object that keeps a set of data values. Most monitored objects are object
+            defined in the WebRTC API; they may be thought of as being internal properties of those
+            objects.
           </p>
         </dd>
         <dt>
-          <dfn>Stats object reference</dfn>
+          <dfn>Stats object</dfn>
         </dt>
         <dd>
-          <p>
-            A stats object has a stable identifier "id". Other stats objects may contain references
-            to this stats object. In a <a>stats value set</a>, these references are represented by
-            a DOMString containing "id" value of the referenced stats object.
-          </p>
-          <p>
-            The stats object is defined by describing the stats value set it returns.
-          </p>
+          This is a set of values, copied out from a monitored object at a specific moment in time.
+          It is recturned as a WebIDL "dictionary" through the GetStats API.
         </dd>
         <dt>
-          <dfn>Stats value set</dfn>
+          <dfn>Stats object reference</dfn>
         </dt>
         <dd>
-          This is a set of values, copied out from a stats object at a specific moment in time. It
-          is recturned as a WebIDL "dictionary" through the GetStats API.
+          <p>
+            A monitored object has a stable identifier "id", which is reflected in all stats
+            objects produced from the monitored object. Stats objects may contain references to
+            other stats objects using this "id" value. In a <a>stats object</a>, these references
+            are represented by a DOMString containing "id" value of the referenced stats object.
+          </p>
         </dd>
         <dt>
           <dfn>Stats value</dfn>
         </dt>
         <dd>
-          Refers to a single value within a stats value set.
+          Refers to a single value within a stats object.
         </dd>
       </dl>
       <p>
-        A stats object changes the values it contains continuously over its lifetime, but is never
-        visible through the GetStats API. A stats value set, once returned, never changes.
+        A monitored object changes the values it contains continuously over its lifetime, but is
+        never visible through the GetStats API. A stats object, once returned, never changes.
       </p>
       <p>
         The stats API is defined in [[!WEBRTC]]. It is defined to return a collection of <a>stats
-        value set</a>s that are a subclass of the RTCStats dictionary. This API is normatively
-        defined in [[!WEBRTC]], but is reproduced here for ease of reference.
+        object</a>s, each of which is of a subclass of the RTCStats dictionary. This API is
```

Comment:
I think the [WebIDL terminology](https://heycam.github.io/webidl/#dfn-inherited-dictionaries) here would be "each of which is a dictionary inherited directly or indirectly from *RTCStats*."